templates/gateway-managed/HotChocolate.Template.Gateway.Managed.csproj

```diff
@@ -8,12 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ChilliCream.Nitro" Version="14.0.0-preview.10" />
-    <PackageReference Include="Microsoft.AspNetCore.HeaderPropagation" Version="8.0.8" />
+    <PackageReference Include="ChilliCream.Nitro.Fusion" Version="14.0.0" />
```

Comment:
is that correct? this are just placeholders that should be replaced in the build process right?

Reply:
You are right